maas/plugins/process_check.py

```diff
@@ -0,0 +1,113 @@
+#!/usr/bin/env python
+#
+# Copyright 2015, Rackspace US, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+import argparse
+import os
+
+from maas_common import metric_bool
+from maas_common import print_output
+from maas_common import status_err
+from maas_common import status_ok
+
+
+def get_process_name(pid):
+    process_path = os.path.join('/proc', pid, 'comm')
+
+    try:
+        process_name = os.readlink(process_path)
```

Comment:
I see you changed to using `/proc/$pid/comm` now but you're still trying to do a `readlink()` call on it, and that doesn't work like `exe` does, as I mentioned last time.